--- config.def.h	2025-07-17 10:12:48.956501106 +0500
+++ config.new.h	2025-07-17 10:20:50.702070471 +0500
@@ -87,8 +87,8 @@
 #define SUPER	Mod4Mask
 
 #define TAGKEYS(KEY,TAG) \
-	{ ALT,			KEY,      view,           {.ui = 1 << TAG} }, \
-	{ ALT|CTRL,		KEY,      toggleview,     {.ui = 1 << TAG} }, \
+	{ ALT,				KEY,      view,           {.ui = 1 << TAG} }, \
+	{ ALT|CTRL,			KEY,      toggleview,     {.ui = 1 << TAG} }, \
 	{ ALT|SHIFT,		KEY,      tag,            {.ui = 1 << TAG} }, \
 	{ ALT|CTRL|SHIFT, 	KEY,      toggletag,      {.ui = 1 << TAG} },
 
@@ -96,10 +96,10 @@
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
+static char rofimon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *roficmd[]	= { "rofi", "-show", "drun", NULL };
 static const char *termcmd[]	= { "sh", "/home/nixos/nixos/scripts/qterm.sh", NULL };
-static const char *nvcmd[]	= { "neovide", NULL };
+static const char *nvcmd[]		= { "neovide", NULL };
 static const char *scrshot[]	= { "flameshot", "gui", NULL };
 static const char *browser[]	= { "chromium", NULL };
 static const char *main_wins[]	= { "/home/nixos/.startup.sh", NULL };
@@ -116,7 +116,7 @@
 	{ SUPER|SHIFT,	XK_b,		spawn,          {.v = browser } },
 	{ ALT|SHIFT,  	XK_p,		spawn,          {.v = main_wins } },
 	{ SUPER|SHIFT,	XK_l,		spawn,          {.v = lockscr } },
-	{ SUPER,		XK_s,		setscheme,      {.i = +1 } },
+	// { SUPER,		XK_s,		setscheme,      {.i = +1 } },
 	
 	// Navigation
 	{ ALT,			XK_space,	spawn,          {.v = roficmd } },
